frontend/src/Components/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import React, { ComponentProps } from 'react';
+import { kinds, sizes } from 'Helpers/Props';
+import styles from './Label.css';
+
+export type LabelProps = {
+  kind?: Extract<(typeof kinds.all)[number], keyof typeof styles>;
+  size?: Extract<(typeof sizes.all)[number], keyof typeof styles>;
+  outline?: boolean;
+} & ComponentProps<'span'>;
```

Comment:
Passes every native props down with their correct typings

Also removes the need for `children` and `title` prop definitions

Reply:
`children` should be required though, currently it's optional.